spec/support/user_sign_in_helper.rb

```diff
@@ -0,0 +1,10 @@
+module UserSignIn
+  def sign_in
+    user = FactoryGirl.create(:user)
+
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.